### ISSUE TITLE
MixedOps-NI-3NReconnect-6k-24m has INSUFFICIENT_TX_FEE errors in HAPIClientValidator

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#1630]
**Summary of the defect**
HAPIClientValidator failed due to the INSUFFICIENT_TX_FEE response code. 

Similar errors are observed in these JRS tests as well: 

MixedOps-NDReconnect-1.5k-25m : https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624267191404800
MixedOps-NIReconnect-1.5k-25m: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624265550401200

This is observed in both 0.15.0 and master branches.

**How to reproduce (if possible)**

0.15 slack report link: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624269598410000

master slack report link: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624265969401600

**Service logs (if applicable)**
```
---- HAPIClientValidator FAILED validation ----
<<ERROR>>
2021-06-21 09:41:02.113 ERROR  38   CustomSpecAssert - Operation 'HapiGetVersionInfo{}' :: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
Exception in thread "thread0" java.lang.IllegalStateException: com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
Caused by: com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
2021-06-21 09:43:49.900 ERROR  38   CustomSpecAssert - Operation 'HapiGetVersionInfo{}' :: Bad answerOnlyPrecheck! expected OK, actual INSUFFICIENT_TX_FEE
...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by tinker-michaelj** [comment]
Fixed by https://github.com/hashgraph/hedera-services/pull/1608 in `master`.

**Comment by ljianghedera** [comment]
@tinker-michaelj are we still able to merge the fix into `release/0.15` after migration testing is done?
